modules/dnn/src/layers/detection_output_layer.cpp

```diff
@@ -844,19 +844,20 @@ class DetectionOutputLayerImpl CV_FINAL : public DetectionOutputLayer
             CV_Assert(numLocClasses == 1);
         }
         locPreds.resize(num);
+        int label;
```

Comment:
Why do we need to declare this variable on this level? It makes sense to reduce variables scope (or use 2 different variables)

Reply:
Modified accordingly, thanks~